openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -317,13 +317,18 @@ def process_response(self, request, response):
 
         if not _is_cookie_marked_for_deletion(request) and _is_cookie_present(response):
             try:
+                is_from_logout = _is_from_logout(request)
```

Comment:
Also, are you disabling the following log message as well?  I didn't see the code for that.
`middleware.py:371 - SafeCookieData user at request '5' does not match user in session: 'None'`
We would need to change it to a Warning level instead of an Error level - since it's an expected case for logouts.